inputs/worker_inner.json

```diff
@@ -0,0 +1,155 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": []
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "args": {
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "bump_release"
+    },
+    {
+      "args": {
+        "labels": "{{IMPLICIT_LABELS}}"
+      },
+      "name": "add_labels_in_dockerfile"
+    },
+    {
+      "name": "change_from_in_dockerfile"
+    },
+    {
+      "name": "add_help",
+      "required": false
+    },
+    {
+      "name": "add_dockerfile"
+    },
+    {
+      "args": {
+        "command": "{{SOURCES_COMMAND}}"
+      },
+      "name": "distgit_fetch_artefacts"
+    },
+    {
+      "args": {
+        "root": "{{KOJI_ROOT}}",
+        "target": "{{KOJI_TARGET}}",
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "koji"
+    },
+    {
+      "args": {
+        "repourls": []
+      },
+      "name": "add_yum_repo_by_url"
+    },
+    {
+      "name": "inject_yum_repo"
+    },
+    {
+      "name": "distribution_scope"
+    }
+  ],
+  "buildstep_plugins": [
+    {
+      "name": "docker_api"
+    }
+  ],
+  "prepublish_plugins": [
+    {
+      "name": "squash"
+    }
+  ],
+  "postbuild_plugins": [
+    {
+      "args": {
+        "image_id": "BUILT_IMAGE_ID"
+      },
+      "name": "all_rpm_packages"
+    },
+    {
+      "name": "tag_by_labels"
+    },
+    {
+      "name": "tag_from_config"
+    },
+    {
+      "name": "tag_and_push",
+      "args": {
+        "registries": {
+          "{{REGISTRY_URI}}": { "insecure": true }
+        }
+      }
+    },
+    {
+      "name": "pulp_push",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "load_exported_image": true,
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "pulp_sync",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "docker_registry": "{{DOCKER_REGISTRY}}",
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "compress",
+      "args": {
+        "load_exported_image": true,
+        "method": "gzip"
+      }
+    },
+    {
+      "name": "pulp_pull",
+      "args": {}
+    }
+  ],
+  "exit_plugins": [
+    {
+      "name": "delete_from_registry",
+      "args": {
+        "registries": {}
+      }
+    },
+    {
+      "name": "koji_promote",
+      "args": {
+        "kojihub": "{{KOJI_HUB}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false,
+        "blocksize": 10485760,
+        "koji_keytab": false,
+        "koji_principal": false
```

Comment:
Which is not a good reason... but it seems that plugin doesn't provide sensible defaults, so I think it's best to leave them in.

```python
    def __init__(self, tasker, workflow, kojihub, url,
                 verify_ssl=True, use_auth=True,
                 koji_ssl_certs=None, koji_proxy_user=None,
                 koji_principal=None, koji_keytab=None,
                 metadata_only=False, blocksize=None,
                 target=None, poll_interval=5)
```